apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/tips/tips.js

```diff
@@ -0,0 +1,79 @@
+
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+function tipDescription( text, conversion, textFallback ) {
+	if ( createInterpolateElement ) {
+		return createInterpolateElement( text, conversion );
+	}
+
+	return textFallback;
+}
+
+export default [
+	{
+		context: 'theme',
+		keywords: [ 'theme', __( 'theme' ) ],
+		description: tipDescription(
+			__(
+				'theme - You can visit the <a>theme directory</a> to select a different design for your site.'
+			),
+			{
+				a: <a href="./themes.php" target="_blank" />,
```

Comment:
Will this work in both wp-admin and Calypso iframe? Same for all the other links..

Reply:
It will work calypsoifying the page. For instance, for `themes` it will redirect to the wp-admin but applying calypsoify.

<img width="300" alt="Screen Shot 2020-05-11 at 6 15 34 AM" src="https://user-images.githubusercontent.com/77539/81561059-f5ec3000-9368-11ea-87fb-adcd063318c6.png">


<img src="https://user-images.githubusercontent.com/77539/81561033-e8cf4100-9368-11ea-83cd-f27384dadc36.png" width="500" />

Wondering if it's possible to get the link according to the current context of the app.

Reply:
currently things like the back button link are passed through the iFrame bridge, so you'd have to do something on the calypso side to pass them through :/

Reply:
It would be important to get them to work in Calypso iframe, so that the experience doesn't feel broken. E.g. you can't drop users to WP Admin plugin page on simple sites since it's blocked from regular users (it works for a12s), but those users can see the Calypso version.